slave/packages.sls

```diff
@@ -102,9 +102,9 @@ runner-packages:
             - libudev-dev:i386
             - libvdpau-dev
             - libvdpau-dev:i386
-            - libvulkan-dev
-            - libvulkan1
-            - libvulkan1:i386
+            - libvulkan-dev: 1.1.70+dfsg1-1
+            - libvulkan1: 1.1.70+dfsg1-1
+            - libvulkan1:i386: 1.1.70+dfsg1-1
```

Comment:
I'm not seeing where the second `:` comes from, is that an apt or salt?

Reply:
a salt thing, I think. I followed this: https://docs.saltstack.com/en/latest/ref/states/all/salt.states.pkg.html#salt.states.pkg.installed

Reply:
that was the documentation I couldn't find :)

looks good to me.